src/EPR.RegulatorService.Frontend.UnitTests/Core/Filters/AbstractSubmissionsFiltersTests.cs

```diff
@@ -30,6 +31,14 @@ public static string GetRandomStatus()
         return statuses[index];
     }
 
+    public static string GetRandomSubmissionPeriod()
+    {
+        string[] submissionPeriods = { "January to June 2023", "July to December 2023", "July to December 2024" };
+        Random random = new Random();
```

Comment:
We need to remove Random from unit tests because sonarqube doesn't like it. Also I think it's better not to have anything that could possibly result in unit tests passing/failing if ran multiple times. You could make this method accept an integer for the index and call it that way to get different records